### ISSUE TITLE
Plaintext is transformed with an emoji

### ISSUE THREAD

**Issue by thany** [vector-im/element-web#22866]
### Steps to reproduce

Best to show pictures.
![image](https://user-images.githubusercontent.com/152227/179324665-4112f68d-a0c6-4607-9ee0-731dbac68024.png)
![image](https://user-images.githubusercontent.com/152227/179324676-07b13fc7-f3de-411f-90d7-67db839a0f50.png)
![image](https://user-images.githubusercontent.com/152227/179324692-a41f8bd5-212b-45ae-bc11-8d89f07ac364.png)
![image](https://user-images.githubusercontent.com/152227/179324713-95e97c71-e1d3-4148-b9bd-10f5f8b64b5d.png)

#### What did you expect?
An emoji is not inserted there. I'm not typing there at all. What's more, `/plain` indicates plaintext. So no emoji. No touching my text.

#### What happened instead?
The opposite, innit.

Need I remind you, this is a form of data loss. Actual real text is lost and replaced by an emoji without an instruction to do so.

### Operating system

Windows 10

### Application version

Element version: 1.11.0 Olm version: 3.2.8

### Will you send logs?

No

**Comment by thany** [comment]
Unless all of `:3 vide` is supposed to be the cat emoji, which I seriously doubt, this issue can be broken down into 2 problems:

1. `/plain` is not respected by inserting emoji.
2. The insertion of that emoji replaces more than the `:3` emoticon it is represented by.

**Comment by aaronraimist** [comment]
There is a setting to turn that off

<img width="650" src="https://user-images.githubusercontent.com/5855073/179339278-641c9681-6db0-4f64-8e64-90283a31ce54.png">

**Comment by SimonBrandner** [comment]
> There is a setting to turn that off
> 
> <img alt="" width="650" src="https://user-images.githubusercontent.com/5855073/179339278-641c9681-6db0-4f64-8e64-90283a31ce54.png">

Yeah, I also think emojis can count as plaintext if the plaintext is in Unicode

**Comment by thany** [comment]
> There is a setting to turn that off
> 
> <img alt="" width="650" src="https://user-images.githubusercontent.com/5855073/179339278-641c9681-6db0-4f64-8e64-90283a31ce54.png">

I dont want to turn off ALL emoji replacements. I just want it to work well.

**Comment by aaronraimist** [comment]
> So what manner of emoji mess will display at the other end, assuming they've not disabled this setting, if this bug continues to exist?

@thany Element doesn't convert received messages. The conversion is only done on the sender's end. So no, if the sender turns that off, the bug does not continue to exist.

**Comment by thany** [comment]
Unless they edit a message, presumably.

Either way, it's still a data loss issue, and should recieve applicable priority.
